sound/soc/intel/boards/sof_maxim_common.c

```diff
@@ -13,6 +13,16 @@
 
 #define MAX_98373_PIN_NAME 16
 
+static const struct snd_kcontrol_new maxim_kcontrols[] = {
```

Comment:
Please describe the purpose of this commit in the commit message. People need to know why we need this commit.